webclient/core/import_data_mappers.py

```diff
@@ -0,0 +1,1522 @@
+        field_mapping["dokument"] = LookupImportField(Dokument)
+        field_mapping["osoba"] = LookupImportField(Osoba)
+        return field_mapping
+
+
+class DokumentPosudekMapper(ImportModelMapper):
+    fields = tuple()
+    model_class = DokumentPosudek
+    primary_key = ("dokument", "posudek")
+
+    @classmethod
+    def get_mapping(cls):
+        field_mapping = super().get_mapping()
+        field_mapping["dokument"] = LookupImportField(Dokument)
+        field_mapping["posudek"] = LookupImportField(Heslar)
+        return field_mapping
+
+
+class TvarMapper(ImportModelMapper):
+    fields = ("id", "poznamka")
+    model_class = Tvar
+    primary_key = "id"
+
+    @classmethod
+    def get_mapping(cls):
+        field_mapping = super().get_mapping()
+        field_mapping["dokument"] = LookupImportField(Dokument)
+        field_mapping["tvar"] = LookupImportField(Heslar)
+        return field_mapping
+
+
+class DokumentCastMapper(ImportModelMapper):
+    fields = ("ident_cely", "poznamka")
+    model_class = DokumentCast
+    primary_key = "ident_cely"
+
+    @classmethod
+    def get_mapping(cls):
+        field_mapping = super().get_mapping()
+        field_mapping["dokument"] = LookupImportField(Dokument)
+        field_mapping["archeologicky_zaznam"] = LookupImportField(ArcheologickyZaznam)
+        field_mapping["projekt"] = LookupImportField(Projekt)
+        return field_mapping
+
+
+class NeidentAkceMapper(ImportModelMapper):
+    fields = ("rok_zahajeni", "rok_ukonceni", "lokalizace", "popis", "poznamka", "pian")
+    model_class = NeidentAkce
+    primary_key = "ident_cely"
+
+    @classmethod
+    def get_mapping(cls):
+        field_mapping = super().get_mapping()
+        field_mapping["dokument_cast"] = LookupImportField(DokumentCast)
+        field_mapping["katastr"] = RuianLookupImportField(RuianKatastr, "kod")
+        return field_mapping
+
+
+class NeidentAkceVedouciMapper(ImportModelMapper):
+    fields = tuple()
+    model_class = NeidentAkceVedouci
+    primary_key = ("neident_akce", "vedouci")
+
+    @classmethod
+    def get_mapping(cls):
+        field_mapping = super().get_mapping()
+        field_mapping["neident_akce"] = LookupImportField(DokumentCast)
+        field_mapping["vedouci"] = LookupImportField(Osoba)
+        return field_mapping
+
+
+class KomponentaMapper(ImportModelMapper):
+    fields = ("ident_cely", "jistota", "presna_datace", "poznamka")
+    model_class = Komponenta
+    primary_key = "ident_cely"
+
+    @classmethod
+    def get_mapping(cls):
+        field_mapping = super().get_mapping()
+        field_mapping["vazba"] = KomponentaLookupImportField(
+            [DokumentacniJednotka, DokumentCast], read_field_name="komponenty"
+        )
+        field_mapping["obdobi"] = LookupImportField(Heslar)
+        field_mapping["areal"] = LookupImportField(Heslar)
+        return field_mapping
+
+
+class KomponentaAktivitaMapper(ImportModelMapper):
+    fields = ()
```

Comment:
Řešeno jinak, než např. ArcheologickyZaznamKatastrMapper - tam jsou všechna pole jmenovaná, tady ne. Proč?

Navíc tady to není stejné ani jako NeidentAkceVedouciMapper, kde je `tuple()`

Reply:
Odebráno, `tuple()` je výchozí hodnota, takže to zde není třeba řešit vůbec.